src/wiki/common-misconceptions.md

```diff
@@ -18,12 +18,30 @@ RAM utilization is also dependent on your RAM capacity - the more RAM you have,
 
 It is commonly thought that Windows is 'bloated' by default and all those pre-installed apps it comes with take up a significant amount of system resources that could be used for other things. This can also apply to system services which people often disable hoping it brings an improvement.
 
-The improvement is not real, if marginal at best. Uninstalling the pre-installed apps, disabling system services, and messing with the registry will 99% of the time **not** bring you an improvement in performance. Windows does not load up all of those apps in the memory at the same time, only when you use them. Services take up a very insignificant amount of memory and CPU resources, even on a low end machine. And Windows does not access the registry sequentially, meaning that deleting registry keys using dubious third party cleaners such as CCleaner will not bring about an improvement as Windows accesses that key directly - it is not akin to disk fragmentation - and no improvement will be gained from similarly dubious registry tweaks.
+The improvement is not real, if marginal at best. Uninstalling the pre-installed apps, disabling system services, and messing with the registry will 99% of the time **not** bring you an improvement in performance. Windows does not load up all of those apps in the memory at the same time, only when you use them. Services take up a very insignificant amount of memory and CPU resources, even on a low end machine. And Windows does not access the registry sequentially, meaning that deleting registry keys using dubious third party cleaners such as CCleaner will not bring about an improvement as Windows accesses that key directly&mdash;it is not akin to disk fragmentation, and no improvement will be gained from similarly dubious registry tweaks.
```

Comment:
I'm not sure I'm following but what's the point of using `&mdash;` instead of the `-` character?

Reply:
Uses — (em dash) instead of -, which is technically the correct form here.